src/model/queries/postNewMemberTechStack.js

```diff
@@ -0,0 +1,24 @@
+const {
+  addMemberTechStack,
+  getTechStackID,
+  getAllTechStack,
+  addTechStack,
+  deleteMemberTech,
+  getMemberTechStack,
+} = require('./query_tech-stack');
+
+
+const postNewMemberTechStack = (memberRepoStack) => {
+  getAllTechStack().then((allTech1) => {
+    const techNames = [];
+    allTech1.forEach(tech => techNames.push(tech.tech));
+    memberRepoStack.forEach(tech => (techNames.includes(tech) ? null : addTechStack(tech)));
+    getAllTechStack().then((allTech2) => {
+      console.lo;
+    });
+  });
+};
+
+postNewMemberTechStack();
+
+module.exports = postNewMemberTechStack;
```

Comment:
Looks like this is unfinished? I've done some work on the SQL queries with processing a tech stack, will have it as a pull request soon.